google/auth/external_account.py

```diff
@@ -389,6 +391,48 @@ def with_token_uri(self, token_uri):
         kwargs.update(token_url=token_uri)
         return self.__class__(**kwargs)
 
+    def introspect_token(self, request):
+        """Introspection of token info.
+
+        If the credential object has a token_info_url provided, we can
```

Comment:
Token_info_url should always be available. not sure if it is worth calling out here. we can just throw an error when it isn't and document that error

Reply:
In the workflow of `introspect_token`, `token_info_url` is always needed and will throw an error.
Do you mean we need to throw error if it's not configured in constructor?
Won't that lost backward compatibility?

Reply:
For `external_account_authorized_user.Credentials` I didn't see the validate of `self._token_info_url` in constructor.
Also I saw the default parameter is `None`. Is that mean some `external_account_authorized_user.Credentials` can skip this field?

Reply:
Yes, looks like when we implemented it to be initializable with minimal info (even though gcloud will always populate token info).